src/martian_apart_hack_sdk/utils.py

```diff
@@ -3,6 +3,7 @@
 import dataclasses
 import json
 from typing import Dict, Any
+from pathlib import Path
```

Comment:
Since we use the Google style guide:
"Use import statements for packages and modules only, not for individual types, classes, or functions."
https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Done.